Logix/Math/Random/RandomDouble.cs

```diff
@@ -0,0 +1,24 @@
+﻿using BaseX;
+using FrooxEngine.LogiX;
+using CodeX;
+using System;
+
+namespace FrooxEngine.Logix.Math
+{
+    [Category("LogiX/Math/Random")]
+    [NodeName("Random Double")]
+    public class RandomDouble : LogixNode
+    {
+        public readonly Input<double> Min;
+        public readonly Input<double> Max;
+        public readonly Output<double> Value;
+
+        protected override void OnEvaluate()
```

Comment:
Random should probably be abstracted elsewhere, we shouldn't be creating new random objects every time we run this node. This also needs the min max swap that the vanilla random nodes have
```cs
    protected override void OnEvaluate()
    {
      float min = Min.EvaluateRaw();
      float max = Max.EvaluateRaw();
      if (min > max)
      {
        float num1 = max;
        float num2 = min;
        min = num1;
        max = num2;
      }
      Value.Value = RandomX.Range(min, max);
    }
```